x/avs/keeper/operator.go

```diff
@@ -0,0 +1,48 @@
+package keeper
+
+import (
+	"fmt"
+	"github.com/ExocoreNetwork/exocore/x/avs/types"
+
+	errorsmod "cosmossdk.io/errors"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k *Keeper) IsExist(ctx sdk.Context, addr sdk.AccAddress) bool {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixAVSOperatorInfo)
+	return store.Has(addr)
+}
+
+func (k *Keeper) SetOperator(ctx sdk.Context, addr string, name string, pub string) (err error) {
+	opAccAddr, err := sdk.AccAddressFromBech32(addr)
+	if err != nil {
+		return errorsmod.Wrap(err, "SetOperator: error occurred when parse acc address from Bech32")
+	}
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixAVSOperatorInfo)
+	operator := &types.OperatorInfo{
+		Name:         name,
+		PublicKey:    pub,
+		IsRegistered: true,
+	}
+	store.Set(opAccAddr, k.cdc.MustMarshal(operator))
+	return nil
+}
+
+func (k *Keeper) GetOperator(ctx sdk.Context, addr string) (operator types.OperatorInfo, err error) {
```

Comment:
Why not get operator in operator module?

Reply:
fixed